client/src/app/metadata-schema-form/metadata-schema-form.module.ts

```diff
@@ -80,7 +80,7 @@ import { VfTabComponent } from './custom/vf-tab/vf-tab.component';
     BaseInputComponent,
     SelectComponent,
     EnumDropDownComponent,
-    SelectListComponent
+    SelectListComponent,
```

Comment:
clean up dangling comma?

Reply:
I actually like dangling commas. Makes it easier to add something to the list. But we should have a consistent style for this

Reply:
I see the benefits of dangling commas, yeah. And I read there's no issue with transpilers/bundlers. Ignore my comment.